integration/commands_administration_test.go

```diff
@@ -140,30 +140,45 @@ func TestCommandsAdministrationCreateDropListDatabases(t *testing.T) {
 
 func TestCommandsAdministrationListDatabases(t *testing.T) {
 	t.Parallel()
-	ctx, collection := setup.Setup(t, shareddata.DocumentsStrings)
-	db := collection.Database()
-	name := db.Name()
+	for _, tc := range map[string]struct {
```

Comment:
@AlekSi my idea was to try different sets of inserted docs and check that stats still works for them. But if it's too much I can leave one case.

Reply:
Well, but we compare with the same numbers, making them even less accurate because the delta should be bigger to accommodate all test cases. For that reason, that's a change for the worse, not for the better.

Let's add different numbers to test cases then. Or just remove them.

Reply:
Fixed